dotcom-rendering/src/components/ExpandableMarketingCard.tsx

```diff
@@ -175,90 +172,98 @@ export const ExpandableMarketingCard = ({
 	setIsClosed,
 }: Props) => {
 	return (
-		<div
-			css={containerStyles}
-			data-component="us-expandable-marketing-card"
-		>
+		<div data-component="us-expandable-marketing-card">
 			<div css={fillBarStyles} />
 			<div css={contentStyles}>
 				{!isExpanded ? (
-					<BannersIllustration type="faded" styles={imageTopStyles} />
-				) : (
 					<>
 						<BannersIllustration
-							type="top"
+							type="faded"
 							styles={imageTopStyles}
 						/>
-						<BannersIllustration
-							type="bottom"
-							styles={imageBottomStyles}
-						/>
-					</>
-				)}
-				<div css={summaryStyles}>
-					<div css={headingStyles}>
-						{heading}
-						<button
+						<section
+							css={summaryStyles}
+							role="button"
+							tabIndex={0}
 							onClick={() => {
-								if (!isExpanded) {
+								setIsExpanded(true);
+							}}
+							onKeyDown={(event) => {
+								if (event.key === 'Enter') {
 									setIsExpanded(true);
-								} else {
+								}
+								if (event.key === 'Escape') {
 									setIsClosed(true);
 								}
 							}}
-							type="button"
-							css={arrowStyles}
 						>
-							{isExpanded ? (
-								<SvgCross />
-							) : (
-								<SvgChevronDownSingle />
-							)}
-						</button>
-					</div>
-					<div css={kickerStyles}>{kicker}</div>
-				</div>
-				{isExpanded && (
-					<div css={detailsStyles}>
-						<div css={sectionStyles}>
-							<h2>We're independent</h2>
-							<p>
-								With no billionaire owner or shareholders, our
-								journalism is funded by readers
-							</p>
-						</div>
-						<div css={sectionStyles}>
-							<h2>We're open</h2>
-							<p>
-								With misinformation threatening democracy, we
-								keep our fact-based news paywall-free
-							</p>
-						</div>
-						<div css={sectionStyles}>
-							<h2>We're global</h2>
-							<p>
-								With 200 years of history and staff across
-								America and the world, we offer an outsider
-								perspective on US news
-							</p>
-						</div>
-						<div css={buttonStyles}>
+							<div css={headingStyles}>
+								<h2>{heading}</h2>
+								<div css={arrowStyles}>
+									<SvgChevronDownSingle />
+								</div>
```

Comment:
Do we want the whole section to be clickable, or just this chevron?

Reply:
I think it would be better for users this way (easier to click as 24x24 for the icon is small, avoid unintentional closing by double-click), but I have an open question with the design team

Reply:
Design are happy with the whole section being clickable